pkg/actions/addon/update.go

```diff
@@ -41,7 +42,7 @@ func (a *Manager) Update(addon *api.Addon, wait bool) error {
 		updateAddonInput.AddonVersion = &summary.Version
 	} else {
 		version := addon.Version
-		if version == "latest" {
+		if version == "latest" || !strings.Contains(version, "eksbuild") {
```

Comment:
this will match on when `version == ""`, is that a problem?

Reply:
actually nvm if i expand the lines above i can see we do something else when it is `""`